### PR TITLE
SQLRowCount support

### PR DESCRIPTION
**Pull request by Koka**
Allows to get affected row count using SQLRowCount function from odbc-sys

### CODE REVIEW

**Comment by Koka** [comment]
For https://github.com/Koka/odbc-rs/issues/80

File: src/handles/hstmt.rs [hunk 1]
```diff
@@ -75,6 +75,13 @@ impl<'env, 'param> HStmt<'env> {
         ret.map(|()| out)
     }
 
+    pub fn affected_row_count(&self) -> Return<SQLLEN> {
+        let mut out: SQLLEN = 0;
+        let ret = unsafe { SQLRowCount(self.handle, &mut out) };
+        let ret: Return<()> = ret.into();
```
Comment (adamreichold) [inline src/handles/hstmt.rs:81]: Purely a style issue, but looks ugly. But I guess `ret.into().map(|()| out)` is not possible due to insufficient type inference?

Comment (Koka) [reply src/handles/hstmt.rs:81]: @adamreichold Unfortunately, yes. It looks good, but causes

> error[E0282]: type annotations needed
>   --> src/handles/hstmt.rs:81:9
>    |         ret.into().map(|()| out)
>    |         ^^^^^^^^^^ cannot infer type for `T`

Comment (Koka) [reply src/handles/hstmt.rs]: @adamreichold Unfortunately, yes. It looks good, but causes

> error[E0282]: type annotations needed
>   --> src/handles/hstmt.rs:81:9
>    |         ret.into().map(|()| out)
>    |         ^^^^^^^^^^ cannot infer type for `T`

**Comment by Koka** [comment]
@pacman82 Could you please release latest changes to crates.io ?

**Pull request merged**